types/object-hash/index.d.ts

```diff
@@ -15,43 +19,22 @@ declare namespace ObjectHash {
 		ignoreUnknown?: boolean;
 		replacer?: (value: any) => any;
 		respectFunctionProperties?: boolean;
-		respectFunctionNames?: boolean;
+        respectFunctionNames?: boolean;
+        respectType?: boolean;
 		unorderedArrays?: boolean;
-		unorderedSets?: boolean;
+        unorderedSets?: boolean;
+        unorderedObjects?: boolean;
 		excludeKeys?: (key: string) => boolean;
 	}
 
-	interface HashTableItem {
-		value: any;
-		count: number;
-	}
-
-	interface HashTableItemWithKey extends HashTableItem {
-		hash: string;
-	}
-
-	export interface HashTable {
```

Comment:
Are you sure, that HashTable is no more available?

Reply:
Yeah :) I checked on both `dist` and `src` code. It hasn't been there since v1.0. It was removed in [this commit](https://github.com/puleos/object-hash/commit/1b52cb176f4925765538aecde8a315402a1a9840#diff-168726dbe96b3ce427e7fedce31bb0bc)